static/js/theme-map/Geo/GeoBounds.js

```diff
@@ -121,10 +121,26 @@ class GeoBounds {
     const newLon = (nw.longitude + this._ne.longitude) / 2 + (this._ne.longitude < nw.longitude ? 180 : 0);
 
     nw.add(-latDist / 2, 0, Unit.DEGREE, projection);
-    nw.longitude = newLon;
+    nw.longitude = this._getNormalizedLongitude(newLon);
 
     return nw;
   }
+
+  /**
+   * Normalize lng to the range [-180, 180]. If you give -181, for
+   * example, we wrap back to 180.
```

Comment:
should -181 wrap back to 179? not a map expert but the math looked a little weird (using mod 361 instead of 360 or 180) and I think that might be why?